cggmp21/src/signing.rs

```diff
@@ -717,11 +717,14 @@ where
 
         tracer.send_msg();
         outgoings
-            .send(Outgoing::p2p(j, Msg::Round1b(MsgRound1b { psi0 })))
+            .feed(Outgoing::p2p(j, Msg::Round1b(MsgRound1b { psi0 })))
             .await
             .map_err(IoError::send_message)?;
         tracer.msg_sent();
```

Comment:
In this round, we send a broadcast message, it should use `feed` as well so we don't flush it before all messages are in the sink

Reply:
https://github.com/LFDT-Lockness/cggmp21/blob/44695389cf1333672c6907f4f298bcd22b0adba3/cggmp21/src/signing.rs#L688-L696